### ISSUE TITLE
fix: build on windows fail on cp

### ISSUE THREAD

**Issue by Policy56** [VeryGoodOpenSource/dart_frog#36]
**Description**

When dart_frog build is run on Powershell on Windows, it's fail.

**Steps To Reproduce**

1. Go to the dart_frog project's
2. Run 'dart_frog build' on powershell
3. See error

**Error**

```
PS C:\Users\*******************\dart_frog_project> dart_frog build
✓ Bundling sources (0.2s)
'cp' n'est pas reconnu en tant que commande interne
ou externe, un programme ex‚cutable ou un fichier de commandes.
```

It means that the command 'cp' is not recognized has internal command.

**Environment**

`flutter doctor -v`

```
[√] Flutter (Channel stable, 3.0.1, on Microsoft Windows [version 10.0.19044.1466], locale fr-FR)
    • Flutter version 3.0.1 at C:\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision fb57da5f94 (5 days ago), 2022-05-19 15:50:29 -0700
    • Engine revision caaafc5604
    • Dart version 2.17.1
    • DevTools version 2.12.2

[√] Android toolchain - develop for Android devices (Android SDK version 32.0.0)
    • Android SDK at C:\Users\Tof\AppData\Local\Android\sdk
    • Platform android-32, build-tools 32.0.0
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[X] Visual Studio - develop for Windows
    X Visual Studio not installed; this is necessary for Windows development.
      Download at https://visualstudio.microsoft.com/downloads/.
      Please install the "Desktop development with C++" workload, including all of its default components

[√] Android Studio (version 2020.3)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[√] VS Code (version 1.67.2)
    • VS Code at C:\Users\Tof\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.40.0

[√] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [version 10.0.19044.1466]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 101.0.4951.54
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 99.0.1150.55

[√] HTTP Host Availability
    • All required HTTP hosts are available

! Doctor found issues in 1 category.
```

**Comment by felangel** [comment]
Thanks for opening an issue! I’ll work on resolving this shortly. Sorry for the inconvenience!

**Comment by felangel** [comment]
This should be resolved by #49 and should be released early tomorrow. Thanks for your patience!

**Comment by felangel** [comment]
This should be fixed in dart_frog_cli v0.0.2-dev.2. You should just need to run `dart pub global activate dart_frog_cli` 🎉 
Let us know if you encounter any other issues and thanks for the patience!
